lightcurve/alcdef.py

```diff
@@ -1,52 +1,63 @@
-from django.contrib import admin
-from django.db import models
-from django.http import HttpResponse
+ALLOWED_KEYWORDS = [
```

Comment:
I would make these `set`s.  Otherwise to check if something is `in` this data structure, python will iterate through the whole list (and set notation will come in handy, per my comment below)

Reply:
Done. I did not reason that performance difference between set and list. Cool!